### DISCUSSION TITLE
Stuck in Full Window

### DISCUSSION THREAD

**Discussion by warplover** [FreeTubeApp/FreeTube#5163] [Q&A] [answered]
I run freetube on a pc. I have kept the program about half the size of my screen for months. I use it all the time and I love it.
I maximized it recently and now it stays that way and I cannot change the size to anything other than max-size. What am I missing?

**Comment by kommunarr** [comment, marked as answer]
Does this issue (#4534) match your description? Try the `F11` key while using FreeTube and see if this addresses your problem.

**Comment by warplover** [comment]
Yes, that worked. Many thanks. That issue does not mention that the video-title bar with the three controls in the right corner disappear (minimize, close, partial-screen). I pulled out ALT-F4 from the memory banks in order to quit the program.
